### DISCUSSION TITLE
Snapshot questions

### DISCUSSION THREAD

**Discussion by Spaceboy60** [SpiralLinux/SpiralLinux-project#496] [Q&A] [answered]
Just wondering how many snapshots are kept by default as I already have 51.  It's there a way to limit the number of snapshots.?

**Comment by geckolinux** [comment]
Hi there, keep in mind that the snapshots only store differential changes, so they might not be taking up all that much space. But at any rate you can customize the automatic snapshot cleanup algorithm with **SnapperGUI (root)** under **Properties** for the `home` and `root` subvolumes, or you can manually edit `/etc/snapper/configs/` .

![image](https://github.com/user-attachments/assets/3521f1ad-f988-478c-ab23-06f470b356c4)

**Reply by Spaceboy60** [reply]
> Hi there, keep in mind that the snapshots only store differential changes, so they might not be taking up all that much space. But at any rate you can customize the automatic snapshot cleanup algorithm with **SnapperGUI (root)** under **Properties** for the `home` and `root` subvolumes, or you can manually edit `/etc/snapper/configs/` .
> 
> ![image](https://private-user-images.githubusercontent.com/2692138/379477872-3521f1ad-f988-478c-ab23-06f470b356c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4MDA3NjksIm5iZiI6MTcyOTgwMDQ2OSwicGF0aCI6Ii8yNjkyMTM4LzM3OTQ3Nzg3Mi0zNTIxZjFhZC1mOTg4LTQ3OGMtYWIyMy0wNmY0NzBiMzU2YzQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjRUMjAwNzQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTJkMDljYWU1NjM3MjhjOTZiMmE3YjdmNmIyODdiYWUyMGRlMjE3OGI0NTM1YTUzNGViYjUxOWFhNDUzZTcxYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.vONGAzxqBGHD2dwcgwMjz1-iYjMVWsgYhxNX38OWggQ)

With regards to snapshots I get the following after running sudo snapper rollback X

Cannot detect ambit since default subvolume is unknown.
This can happen if the system was not set up for rollback.
The ambit can be specified manually using the --ambit option

**Reply by geckolinux** [reply, marked as answer]
Hi there, as mentioned [previously](https://github.com/orgs/SpiralLinux/discussions/478), SpiralLinux is not designed to work with `snapper rollback`, but rather with `snapper-rollback` 
https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks
